README.rst

```diff
@@ -24,27 +24,26 @@ Here is an overview of what the API looks like:
 
 .. code-block:: python
 
-    from kinto_client import Bucket
+    from kinto_client import Client
 
-    bucket = Bucket('default', server_url='http://localhost:8888/v1',
+    client = Client(server_url="http://localhost:8888/v1",
                     auth=('alexis', 'p4ssw0rd'))
-    todo = bucket.get_collection('todo')
-
-    records = todo.get_records()
+    records = client.get_records(bucket='default', collection='todos')
     for i, record in enumerate(records):
         record.data.title = 'Todo #%d' %i
 
-    todo.save_records(records)
+    client.update_records(records)
```

Comment:
it's the one that I though about first. I find update clearer in the meaning, but they are very similar.